theano/gof/cc.py

```diff
@@ -235,6 +235,7 @@ def struct_gen(args, struct_builders, blocks, sub):
 
     # declares the storage
     storage_decl = "\n".join(["PyObject* %s;" % arg for arg in args])
+    storage_init = "\n".join(["this->%s = NULL;" % arg for arg in args])
```

Comment:
Forget my comments, it is for the storage, not for the param. See @abergeron comments. I'll delete mine.